Makefile

```
@@ -0,0 +1,76 @@
+.POSIX:
+
+.PHONY: help registries test-sync test-scorer lint-app test-app lint-shell \
+	test-shell smoke acceptance-scorer acceptance-quiz-only \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)